### DISCUSSION TITLE
If ideahub is a webapp, could someone please send me the url ?

### DISCUSSION THREAD

**Discussion by lmponcio** [MakeContributions/ideahub#126] [Q&A] [answered]
Hello!
Thanks for allowing me to join this community

Is there a url that I can use for accessing a webapp where all the ideas of ideahub are listed?
If it exists, could someone please copy it here so I can access it? (I just want to see  all the ideas in a nice format instead of scrolling through the json files)

Apologies in advance if the question is a bit silly :)
Thanks 
Regards

**Comment by ming-tsai** [comment, marked as answer]
HI @lmponcio,
Here is the link, and you could see it on the repo.
<img width="895" alt="image" src="https://user-images.githubusercontent.com/37890026/196048992-6b427ff1-5bab-4f6f-97cc-5c8d5c8c37d1.png">

**Reply by lmponcio** [reply]
Thanks a lot @ming-tsai , apologies I didn't see it
